src/renderer/components/avatar/avatar.tsx

```diff
@@ -21,70 +21,20 @@
 
 import styles from "./avatar.module.css";
 
-import React, { HTMLAttributes, ImgHTMLAttributes } from "react";
+import React, { HTMLAttributes } from "react";
 import randomColor from "randomcolor";
-import GraphemeSplitter from "grapheme-splitter";
-import { cssNames, iter } from "../../utils";
+import { cssNames } from "../../utils";
 
 export interface AvatarProps extends HTMLAttributes<HTMLElement> {
```

Comment:
These changes to `<Avatar />` were made to make it more reusable. Namely, I felt that there was too much business logic going on in here. Namely the computation of `shortName`'s.

Futhermore, because only `src` was a field (and not icon). There was a strange disconnect where the use of `<Avatar>` would include both passing in an optional `src` as well as optionally rendering an `<Icon />`. 

For that reason, I made `<EntityIcon>` to do all the work instead.